src/foam/java/refinements.js

```diff
@@ -55,6 +55,7 @@ foam.CLASS({
         propName: this.name,
         propType: this.javaType,
         propRequired: this.required,
+        of: this.of,
```

Comment:
So this should be added to separate abstractPropertyInfos such as `abstractFObjectPropertyInfo` ?